WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md

```diff
@@ -13,9 +13,9 @@ ms.technology: identity-adds
 ---
 # Enabling Clients to Locate the Next Closest Domain Controller
 
->Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012
+>Applies To: Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2012, Windows Server 2008 R2, Windows Server 2008
 
-If you have a domain controller that runs  Windows Server 2008  or  Windows Server 2008 R2 , you can make it possible for client computers that run Windows Vista, Windows 7,  Windows Server 2008 , or  Windows Server 2008 R2  to locate domain controllers more efficiently by enabling the **Try Next Closest Site** Group Policy setting. This setting improves the Domain Controller Locator (DC Locator) by helping to streamline network traffic, especially in large enterprises that have many branch offices and sites.  
+If you have a domain controller that runs Windows Server 2008 and above, you can make it possible for client computers that run Windows Vista and above or Windows Server 2008 and above to locate domain controllers more efficiently by enabling the **Try Next Closest Site** Group Policy setting. This setting improves the Domain Controller Locator (DC Locator) by helping to streamline network traffic, especially in large enterprises that have many branch offices and sites.  
```

Comment:
Windows Server 2008 and above > Windows Server 2008 or newer

Reply:
done throughout the document